.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -11,60 +11,78 @@ body:
 - type: checkboxes
   attributes:
-    label: Before proceeding, check to make sure there isn’t an existing issue for this bug.
-    description:
+    label: Before proceeding, make sure there isn’t an existing issue for this bug.
+    description: Check the [Issue Tracker](https://github.com/thegetty/quire/issues) to avoid duplicate issues.
     options:
     - label: I have searched the existing issues and determined this is a new bug.
       required: true    
+- type: textarea
+  id: expected
+  attributes:
+    label: Expected Behavior
+    description: Describe what you expected to happen had everything operated accordingly. 
+  validations:
+    required: true
+- type: textarea
+  id: actual
+  attributes:
+    label: Actual Behavior
+    description: Describe what happened.
+  validations:
+    required: true
 - type: textarea
   id: reproduce
   attributes:
-    label: Steps to reproduce
+    label: Steps to Reproduce
     description: Describe what steps you took that led to the issue. Please be as specific as possible.
+    placeholder: Step 1... Step 2... Step 3... 
   validations:
     required: true
 - type: textarea
-  id: actual
+  id: quire-version 
   attributes:
-    label: Actual behavior
-    description: Describe what happened and how it diverged from what was anticipated.
+    label: Quire Version Numbers
+    description: Run `quire --version` to get the CLI version number. In your project, open the `package.json` file to find the Project Package (quire-11ty) version number. 
+    placeholder: 
   validations:
     required: true
 - type: textarea
-  id: expected
+  id: node-version 
   attributes:
-    label: Expected behavior
-    description: Describe what you expected to happen had everything operated accordingly. 
+    label: Node Version Number
+    description: Run `node --version` to get the Node version number. 
+    placeholder: 
   validations:
     required: true
 - type: textarea
-  id: version 
+  id: OS
   attributes:
-    label: Version numbers
-    description: Providing version numbers will help us diagnose the issue more quickly. Run `quire --version` to get the Quire version number and `node --version` to get the Node version number. 
-    placeholder: Please include operating system (macOS/Windows), operating system version, Quire version, Node version, and any other version numbers relevant to the issue.
+    label: Operating System
+    description: What Operating System you are working on? Please include the version number.
   validations:
     required: true
-- type: dropdown
+- type: textarea
   id: browsers
   attributes:
-    label: What browsers are you seeing the problem on?
-    multiple: true
-    options:
-    - Firefox
-    - Chrome
-    - Safari
-    - Microsoft Edge
+    label: Web Browser 
+    description: What web browser you are working on (Chrome, Safari, Firefox, Microsoft Edge)? Please include the browser version number.
+  validations:
```

Comment:
I don't think browser needs to be required, for example with a CLI bug the browser wouldn't be relevant

Reply:
great feedback @hbalenda. I've made that correction!